### PR TITLE
Suggested mods

### PR DESCRIPTION
**Pull request by leannep**
Updates with oldtext/newtext macros

### CODE REVIEW

File: dm10.tex [hunk 1]
```diff
@@ -5,25 +5,29 @@ \section{DM10: Science Platform Portal}\label{sect:dm10}
...
 At this point, we would take the opportunity to evaluate the then-available technology for the Portal.
 % The portal work started too early on LSST by many estimations.
 
 The LSST EPO subsystem also needs a portal and has already decided that
 Firefly is not a suitable technology. A consideration of possible alternatives
 is already underway.
 Within the AURA family, STScI and NOAO have portals.
-We have concerns about their capabilities and usability, but perhaps those could be addressed with appropriate directives from AURA and synergy with LSST.
+We have concerns about their capabilities and usability
+\newtext{and believe that the current version of the LSST Portal is more advanced and suited to Science goals of LSST},
```
Comment (jdswinbank) [inline dm10.tex]: no need to capitalize “science”

File: dm10.tex [hunk 2]
```diff
@@ -5,25 +5,29 @@ \section{DM10: Science Platform Portal}\label{sect:dm10}
...
+but perhaps those could be addressed with appropriate directives from AURA and synergy with LSST.
 
 The existing system is adequate for the analysis of commissioning data.
 We would therefore target the availability of DR1 for the launch of an updated
 portal.
-The other area of potential concern is the front end for the alert subscription in the mini broker.
-This should be thin veneer on the mini broker API and is also not needed until start of operations.
+The other area of potential concern is the front end for the ale	rt subscription in the mini broker.
```
Comment (jdswinbank) [inline dm10.tex]: Spacing is weird.

**Review by jdswinbank — APPROVED** [review]
👍

**Comment by womullan** [comment]
Leanne I think in future its not necessary to do the old text new text .. the diff shows it well enough ..

**Comment by timj** [comment]
We use oldtext/newtext for LCRs where we don't expect people to be wanting to look at git history.

**Pull request merged**